libretroshare/src/retroshare/rsgxsifacetypes.h

```diff
@@ -34,10 +34,7 @@
 #include "serialiser/rstypeserializer.h"
 #include "util/rstime.h"
 
-typedef GXSGroupId   RsGxsGroupId;
```

Comment:
mmmh. The proper name is RsGxsGroupId, it's not GXSGroupId. The later was some kind of intermediate proxy and should be the one to remove. 99.99% of the code uses RsGxsGroupId.

Reply:
Yeah I agree the one I have removed is `GXSGroupId` despite to what looking at this line seems to suggest ;). I have named it `RsGxsGroupId` when it is defined instead of having that alias around, no no one get tempted to use `GXSGroupId` and `follow symbol under cursor` hop directly to the definition instead of having to jump over 3 file to get to the real type :p